python_modules/dagster-airflow/dagster_airflow/scaffold.py

```diff
@@ -53,21 +53,8 @@ def _split_lines(lines):
     return (lines.strip('\n') + ',').split('\n')
 
 
-def _key_for_marshalled_result(step_key, result_name, prepend_run_id=True):
-    '''Standardizes keys for marshalled inputs and outputs.'''
-    return (
-        '{tmp}'
-        + '{sep}'
-        + ('{run_id_prefix}' if prepend_run_id else '')
-        + _normalize_key(step_key)
-        + '___'
-        + _normalize_key(result_name)
-        + '.pickle'
-    )
-
-
-def _step_executions_key(solid_name):
-    return 'STEP_EXECUTIONS_' + solid_name.upper()
+def _steps_for_key(solid_name):
```

Comment:
whats going on here

Reply:
just renaming this since "step_executions" is going away